ArduPlane/quadplane.cpp

```diff
@@ -1178,10 +1187,28 @@ void QuadPlane::control_loiter()
     // run loiter controller
     loiter_nav->update();
 
-    // nav roll and pitch are controller by loiter controller
+    // nav roll and pitch are controlled by loiter controller
     plane.nav_roll_cd = loiter_nav->get_roll();
     plane.nav_pitch_cd = loiter_nav->get_pitch();
 
+    // avoid flying backwards at high speed
+    Vector3f vel_ned;
```

Comment:
I'd probably recommend introducing an extra block of `{`'s around all of this, that way you don't leak the vel_ned/airspeed symbols to the rest of the function, which won't have access to checking the return values from the if statement.

Reply:
I did this by enclosing the code in a >0 check on loit_rev_spd